sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -48,8 +48,6 @@ internal sealed class KnownType
         internal static readonly KnownType NUnit_Framework_TestCaseSourceAttribute = new KnownType("NUnit.Framework.TestCaseSourceAttribute");
         internal static readonly KnownType NUnit_Framework_TestFixtureAttribute = new KnownType("NUnit.Framework.TestFixtureAttribute");
         internal static readonly KnownType NUnit_Framework_TheoryAttribute = new KnownType("NUnit.Framework.TheoryAttribute");
-        internal static readonly KnownType Oracle_ManagedDataAccess_Client_OracleCommand = new KnownType("Oracle.ManagedDataAccess.Client.OracleCommand");

```

Comment:
Dependencies were not removed when we removed the old implementation of the rule.